cardano-api/src/Cardano/Api/Block.hs

```diff
@@ -156,7 +191,10 @@ fromConsensusBlock CardanoMode =
 -- Block headers
 --
 
-data BlockHeader
+data BlockHeader = BlockHeader !SlotNo
+                               !(Cardano.Api.Hash.Hash Cardano.Api.Block.BlockHeader)
```

Comment:
Do we need this module qualification? We usually have API things unqualified, and use module-qualified names for consensus & ledger types.

Reply:
Just some vestigial annotations from my own working out. Will remove.